Cargo.toml

```diff
@@ -27,7 +27,7 @@ anyhow = "1"
 assert_matches = "1"
 async-trait = "0.1"
 aws-lc-rs = "1.13.0"
-backoff = "0.4.0"
+backon = "1.5.0"
```

Comment:
Do we ever not enable feature `tokio-sleep`? Is it easier to enable the feature up here and remove it from the other workspace `Cargo.toml`s?

Reply:
Fixed in 6092c1d384690e438efc4b4021ec06f5bac063ed